doc/colvars-refman-main.tex

```diff
@@ -6489,6 +6489,383 @@
+
+The implementation of \texttt{opes{\textunderscore}metad} and its documentation are largely based on the \href{https://www.plumed.org/doc-master/user-doc/html/_o_p_e_s__m_e_t_a_d__e_x_p_l_o_r_e.html}{OPES module} of the \href{https://www.plumed.org/}{PLUMED} package.
+
+
+\cvsubsubsec{Implementation notes}{sec:colvarbias_opes_notes}
+The \texttt{opes{\textunderscore}metad} in Colvars is largely based on the code of the OPES module in PLUMED \cite{tribello_plumed_2014}, except that it has the following limitations:
+\begin{itemize}
+\item Colvars does not support chaining bias actions in PLUMED, so the Colvars OPES implementation cannot take the biasing energies of other biases, such as harmonic restraints, into account.
+\item Due to the same limitation of Colvars as described above, you cannot collect samples on-the-fly by chaining \texttt{opes{\textunderscore}metad} with a weighted histogram for PMF calculations. Although the PMF calculation via reweighting is implemented into the \refkey{pmf}{opes_metad|pmf} option, the functionality is quite limited. The CVs in \refkey{pmfColvars}{opes_metad|pmf_colvars} must be among the biased ones. In other words, you cannot apply bias on $(\xi_1, \xi_2)$ and compute the PMF along $\xi_3$ by the on-the-fly reweighting using \refkey{pmf}{opes_metad|pmf} (you can only compute the PMF along $\xi_1$, $\xi_2$ or $(\xi_1, \xi_2)$). In addition, you can only specify one set of CVs for estimating the PMF (you cannot apply the OPES bias on $(\xi_1, \xi_2, \xi_3)$ and compute two 2D PMFs along $(\xi_1, \xi_2)$ and $(\xi_2, \xi_3)$ simultaneously using the \refkey{pmf}{opes_metad|pmf} option). However, you can always bypass this limitation by post-hoc reweighting with the trajectories.
+\item Colvars computes the biases in parallel with SMP, so it cannot use multiple threads again for computing the probability and the normalization factor, searching the kernels that are able to compress and updating the neighbor list. As a result, the CPU performance of this implementation may be slower than the PLUMED OPES module.
+\end{itemize}
+
+The following table summarizes the option names in Colvars and the corresponding option names in the PLUMED OPES module:
+\begin{table}[h]
+\centering
+\begin{tabular}{ll}
+\textbf{Colvars keyword} & \textbf{PLUMED keyword} \\
+\hline
+barrier & BARRIER \\
+newHillFrequency & PACE \\
+gaussianSigma & SIGMA \\
+gaussianSigmaMin & SIGMA\_MIN \\
+kernelCutoff & KERNEL\_CUTOFF \\
+compressionThreshold & COMPRESSION\_THRESHOLD \\
+adaptiveSigma & (use SIGMA=ADAPTIVE) \\
+adaptiveSigmaStride & ADAPTIVE\_SIGMA\_STRIDE \\
+neighborList & NLIST \\
+neighborListNewHillReset & NLIST\_PACE\_RESET \\
+neighborListParameters & NLIST\_PARAMETERS \\
+noZed & NO\_ZED \\
+fixedGaussianSigma & FIXED\_SIGMA \\
+recursiveMerge & (the opposite of RECURSIVE\_MERGE\_OFF) \\
+calcWork & CALC\_WORK
+\end{tabular}
+\end{table}
+
+\cvsubsubsec{General options}{sec:colvarbias_opes_general_options}
```

Comment:
If `FILE` is not needed because Colvars restarts from a single checkpoint file, then this could be maybe clarified somewhere?

Reply:
Done. (see https://github.com/Colvars/colvars/pull/709/commits/3dec310eb708e7b9cc61d6143e68d3d2c4ee4382)